doc/en/Makefile

```diff
@@ -13,7 +13,6 @@ PAPEROPT_letter = -D latex_paper_size=letter
 ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 
 REGENDOC_ARGS := \
-	--normalize "/in \d+.\d+ seconds/in 0.12 seconds/" \
```

Comment:
for better or worse - i am responsible for that one, to be fully in sync a regendoc run would be appreciated

Reply:
Removing the normalization means that now every regendoc run will produce a lot of noise, because the times will most likely be different in each run. I'm curious what's the rationale for this change? 🤔

Reply:
Translating all runtimes to be `0.12 seconds` will document false-y examples on time-sensitive features like cache. If we want to keep this normalization, we'll have to re-word or remove example(s) in cache documentation.
The purpose of this change is that we don't limit what can be documented in terms of examples.

Reply:
Thanks @hyunchel for the reply. Can you point an example of a time-sensitive example?

Reply:
@nicoddemus  sorry about that, I will keep git commits next time as a reference.
Here is one example that cause this entire conversation: https://docs.pytest.org/en/latest/cache.html?highlight=cache#the-new-config-cache-object

Reply:
we ought to investigate how to make regendoc more flexible there, i'll put it down as a experiment for the Christmas holiday

Reply:
@hyunchel I see, thanks!

If that's the only example, I suggest we change that `time.sleep()` call to something like:

```python
@pytest.fixture
def mydata(request):
    val = request.config.cache.get("example/value", None)
    if val is None:
        expensive_computation()
        val = 42
        request.config.cache.set("example/value", val)
    return val
```

I really would like to avoid removing the time normalization as this will generate tons of noise for each regendoc.

We can then use @RonnyPfannschmidt's experiment when that comes around to improve that specific example.

Reply:
I just pushed a commit with a regendoc run - I should have checked the comments first :/
Once I remove the commit, I'll push a new commit to change that `time.sleep()`. Thanks!

Reply:
I have reverted previous commits and updated the cache example with a print statement.

@nicoddemus there are other examples related to time. Things like test duration not matching with `0.12 seconds` normalization(e.g. https://docs.pytest.org/en/latest/example/simple.html#profiling-test-duration) is a pet peeve for me, but I didn't submit a change for it. Hopefully normalization gets a vamp during Christmas time.

@RonnyPfannschmidt I will push with a full regendoc sync when I have an access to a linux machine.